### ISSUE TITLE
[ENHANCEMENT] Accessibility issue with role="link"

### ISSUE THREAD

**Issue by didierrob** [pnp/custom-learning-office-365#703]
**Is your feature request related to a problem? Please describe.** 

There is an accessibility issue where the square banners use a div with a link role.
The problem with the ```<div role="link">``` is that a keyboard-only user will not be able to click these and since they are coded with a link role, a screen reader will see them as links. Usually, a link goes to another page but since those banners open up a new content on the same page, the use of a button would be more appropriate.

**Describe the solution you'd like**
The solution would be to use a``` <button>``` tag if possible.
This would remove that problem and would come with a lot of accessibility features that would help users using screen reader.

**Describe alternatives you've considered**
Another alternative would be to change some elements and add others: 
1. the role would need to be changed from links to button.
2. a ```tabindex = "0"``` would need to be added so it is possible to tab to the section
3. a click handler would need to be configured to check for "Enter" key or "Space" key to activate
4. a focus indicator would need to be created to show when a banner is selected

**Additional context**
Here a sample of the code: 
```
<div role="link"> 
  <div> 
     <img src="someImage.png"> 
  </div> 
  <div>Product Reading List</div> 
<div> 
```
![image](https://user-images.githubusercontent.com/108026885/197052131-7824c53b-5911-48b0-be41-b137ade13cbf.png)

**Comment by dcashpeterson** [comment]
@didierrob Thanks for bringing this to our attention. We will have a look at it.

**Comment by StfBauer** [comment]
Thank @didierrob you we incorporated your recommendation in the current v5 beta version.

**Comment by dcashpeterson** [comment]
@didierrob we have just released the V5 beta #801. We welcome you to participate in our V5 beta testing if you would like.
